bzl/host/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+package(default_visibility = ["//visibility:public"])
+
+config_setting(
+    name = "linux",
+    constraint_values = [
+        "@platforms//os:linux",
+    ]
+)
+
+config_setting(
+    name = "linux_x64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:x86_64",
+    ]
+)
+
+config_setting(
+    name = "macos",
+    constraint_values = ["@platforms//os:macos"]
+)
+
+config_setting(
```

Comment:
I think we explicitly don't support 32-bit architectures at the moment. Can this be commented out, so that we don't accidentally use it until we have built and tested that support?